File: DemoServer/AdditionalLanguages/python/basics/deleteDocument/deleteDocument.py
from demo_example import Example, RunParamsBase

class RunParams(RunParamsBase):
    def __init__(self, document_id: str):
        self.document_id = document_id

class DeleteDocument(Example):
    def run(self, run_params: RunParams) -> None:
        documentID = run_params.document_id

        #region Demo
        with self.document_store_holder.store().open_session() as session:
            #region Step_1
            session.delete(documentID)
            #endregion

            #region Step_2
            session.save_changes()
            #endregion
        #endregion

File: DemoServer/AdditionalLanguages/php/basics/deleteDocument/DeleteDocument.php
<?php

namespace RavenDB\Demo\basics\deleteDocument;

//region Usings
use RavenDB\Documents\Session\DocumentSessionInterface;
//endregion
use RavenDB\Demo\common\DocumentStoreHolder;

class DeleteDocument
{
    public function __invoke(RunParams $runParams)
    {
        $documentID = $runParams->getDocumentId();

        //region Demo
        $session = DocumentStoreHolder::getStore()->openSession();
        try {
            //region Step_1
            $session->delete($documentID);
            //endregion

            //region Step_2
            $session->saveChanges();
            //endregion
            
        } finally {
            $session->close();
        }
        //endregion
    }
}
